CONTRIBUTING.md

```diff
@@ -0,0 +1,115 @@
+# Contributing to Fleet
+
+Fleet accepts contributions via GitHub issues and pull requests.
+This document outlines the process to get your pull request accepted.
+
+## Start With An Issue
+
+Prior to creating a pull request it is a good idea to [create an issue].
+This is especially true if the change request is something large.
+The bug, feature request, or other type of issue can be discussed prior to
+creating the pull request. This can reduce rework.
+
+[Create an issue]: https://github.com/rancher/fleet/issues/new
+
+## Sign Your Commits
+
+A sign-off is a line at the end of the explanation for a commit.
+All commits have to be signed. Your signature certifies that you wrote the patch
```

Comment:
Are we expecting all commits to be signed?

Reply:
where would we draw the line otherwise ?